internal/cri/opts/spec_opts.go

```diff
@@ -118,6 +118,29 @@ func WithAnnotation(k, v string) oci.SpecOpts {
 	}
 }
 
+// WithAffinityCPUs sets the CPU affinity values in runtime spec for windows.
+func WithAffinityCPUs(affinityCPUs []*runtime.WindowsCpuGroupAffinity) oci.SpecOpts {
```

Comment:
Curious if we also want to make this feature available for non-CRI purposes; in that case we could (as I mentioned in an earlier comment) implement the core functionality in the `oci` package, and use this as an adaptor; something like (very quick draft);
https://github.com/containerd/containerd/commit/e9c0018681db7313ee2ab5a22f47bc25d15e6cb8

Reply:
I don't think we have plans to support this feature outside of CRI for windows. So I think it would be good to keep it in internal/cri/*